static/translations/messages_en.json

```diff
@@ -211,5 +211,9 @@
   "🚌 Commuting": "🚌 Commuting",
   "🤒 Out sick": "🤒 Out sick",
   "🌴 Vacationing": "🌴 Vacationing",
-  "🏠 Working remotely": "🏠 Working remotely"
+  "🏠 Working remotely": "🏠 Working remotely",
```

Comment:
one more nit: please add somewhere other than the end of this list. That way there isn't the extra removed/added line in the diff from the added comma.

(The root problem here is with JSON's syntax, but in this context it's easy to work around.)